examples/nidaqmx/analog-input.py

```diff
@@ -73,10 +72,16 @@ def RaiseIfError(response):
 
     RaiseIfError(client.StartTask(nidaqmx_types.StartTaskRequest(task=task)))
 
+    response = client.GetTaskAttributeUInt32(
```

Comment:
Do you think it's worth adding a comment here that this isn't necessary for this example (since we know there's only one channel), but just a demonstration of how to do it?

Reply:
Yes, I think so. Good idea.